SETUP/tests/jsTests/formatPreviewTests.js

```diff
@@ -0,0 +1,113 @@
+/* global QUnit analyse */
+
+QUnit.test("Format preview test", function( assert ) {
+    let inLineTags = "[ibfg]|sc";
```

Comment:
const since we don't seem to modify this. What is this? Should it come from the preview.js code?

Reply:
I agree. This is not really the right way do do it. I'll reconsider this.